package.json

```diff
@@ -54,7 +54,7 @@
   },
   "bin": "cli.js",
   "scripts": {
-    "lint": "eslint katex.js katex.webpack.js cli.js webpack.common.js webpack.config.js webpack.dev.js src test contrib dockers && stylelint static/fonts.less static/katex.less",
```

Comment:
I'll make a separate PR to https://github.com/KaTeX/katex-fonts to lint fonts.less now that it's living in that repo.